mutiny-wasm/src/utils.rs

```diff
@@ -27,16 +27,15 @@ pub fn now() -> Duration {
         .unwrap()
 }
 
-#[allow(dead_code)]
 pub async fn sleep(millis: i32) {
-    let mut cb = |resolve: js_sys::Function, _reject: js_sys::Function| {
-        web_sys::window()
-            .unwrap()
-            .set_timeout_with_callback_and_timeout_and_arguments_0(&resolve, millis)
-            .unwrap();
-    };
-    let p = js_sys::Promise::new(&mut cb);
-    wasm_bindgen_futures::JsFuture::from(p).await.unwrap();
+    #[cfg(target_arch = "wasm32")]
+    {
+        windowless_sleep::sleep(millis).await;
+    }
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        tokio::time::sleep(Duration::from_millis(millis.try_into().unwrap())).await;
+    }
```

Comment:
This should only be targetting wasm32, so I'm not sure why tokio stuff is in here.

Reply:
we should be able to just get rid of this and re-export from mutiny core